src/collector.ts

```diff
@@ -132,6 +161,37 @@ class NaiveGomodParser {
             return dependencies;
         }, []);
 
+        function checkReplaceModule(dep: IDependency) {
+            let replaceDependency = replaceDependencies.get(dep.name.value);
```

Comment:
This will not work as commented before, map cannot have package name as key because there can be multiple replace statement with same package name and different versions.